Chiazokam/iReporterApp
# #162367771 create the endpoint to delete a redflag

#### What does this PR do?
- implements tests for the delete redflag api endpoint
- implements the api endpoint

#### Description of Task to be completed?
- write tests for the delete intervention api endpoint
- implement the api endpoint

#### How should this be manually tested?
- make sure to have node installed on ur pc
- cd into the desired destination
- clone repo into the folder
- cd into the iReporterApp folder
- run ` npm run test`
- the test results are then seen

#### What are the relevant pivotal tracker stories?
[#162367771](https://www.pivotaltracker.com/story/show/162367771)

#### Screenshots (if appropriate)
Below is a sample of a returned data
![image](https://user-images.githubusercontent.com/26940294/49644146-ee0fba00-fa17-11e8-847b-ef17f7061298.png)
